src/app/MessageDef/TimedRequest.h

```diff
@@ -0,0 +1,89 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+#include <app/AppBuildConfig.h>
+#include "Builder.h"
+#include "Parser.h"
+#include <app/util/basic-types.h>
+#include <core/CHIPCore.h>
+#include <core/CHIPTLV.h>
+#include <support/CodeUtils.h>
+#include <support/logging/CHIPLogging.h>
+
+namespace chip {
+namespace app {
+namespace TimedRequest {
+enum
+{
+    kCsTag_Timeout    = 0,
```

Comment:
Is there a unit of time associated with timeouts?

Wondering if we should name things like `kCsTag_TimeoutSec` and `TimedRequest::Builder::TimeoutSec` and similar

Reply:
Timeouts are in milliseconds.

Reply:
Done, thanks